src/NuGet.Clients/NuGet.PackageManagement.UI/Models/IconBitmapStatus.cs

```diff
@@ -10,15 +10,10 @@ public enum IconBitmapStatus
         NeedToFetch,
         Fetching,
         MemoryCachedIcon,
-        DownloadedIcon,
-        EmbeddedIcon,
+        FetchedIcon,
         DefaultIcon,
         DefaultIconDueToDecodingError,
         DefaultIconDueToNullStream,
-        DefaultIconDueToNoPackageReader,
-        DefaultIconDueToNetworkFailures,
-        DefaultIconDueToWebExceptionBadNetwork,
-        DefaultIconDueToWebExceptionOther,
```

Comment:
You're reverting these statuses for what reason?

Reply:
followed up on earlier feedback to move from webclient to httpclient which changes which errors i am able to detect, and moved to put it behind a service function call, which make it trickier to communicate granularity.